Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -298,8 +304,12 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
     /// Prose test 1 of change stream spec.
     /// "ChangeStream must continuously track the last seen resumeToken"
     func testChangeStreamTracksResumeToken() throws {
-        guard MongoSwiftTestCase.topologyType != .single else {
-            print(unsupportedTopologyMessage(testName: self.name))
+        let testRequirements = TestRequirement(
+            acceptableTopologies: ["replicaSet", "sharded"].map { TestTopologyConfiguration(from: $0) })
```

Comment:
here is where the linter is complaining I think. If you move the last `)` on line 308 to its own line, I think it should be good to go.

Reply:
did this for all similar places also